packages/stockflux-launcher/src/popups/PopupWindow.js

```diff
@@ -0,0 +1,82 @@
+import React, { useState, useEffect, useCallback } from 'react';
+import Components from 'stockflux-components';
+import useChildWindow from '../search-results/helpers/useChildWindow';
+
+const ModalPopup = ({ message, options, children }) => {
```

Comment:
Filename does not match component name.

Reply:
Changed component name to match filename.